lib/openstudio-standards/standards/necb/NECB2015/hvac_systems.rb

```diff
@@ -125,18 +125,18 @@ def apply_maximum_loop_pump_power(model)
             max_powertoload = model_find_object(max_total_loop_pump_power_table, search_hash)['total_normalized_pump_power_wperkw']
           when 'OS_Pump_VariableSpeed'
             pumps << supplycomp.to_PumpVariableSpeed.get
-            total_pump_power += model.getAutosizedValue(supplycomp, 'Design Power Consumption', 'W').to_f
+            total_pump_power += supplycomp.getAutosizedValue('Design Power Consumption', 'W').to_f
```

Comment:
why not `supplycomp.ratedPowerConsumption`?

Reply:
good call, changed